uber/decorators.py

```diff
@@ -234,6 +233,10 @@ def with_restrictions(*args, **kwargs):
                 if not cherrypy.session.get('staffer_id'):
                     raise HTTPRedirect('../signups/login?message=You+are+not+logged+in')
 
```

Comment:
Unless I'm mis-reading this, it doesn't actually whether whether someone has the appropriate access.  So like, if someone tried to access a page with this restriction and we WERE at the con, it would just always allow it even if the person wasn't even logged in.

I'll need to check this to make sure, but I'm pretty sure that's what's happening here.